### DISCUSSION TITLE
send a list of rejected email to the original recipients ?

### DISCUSSION THREAD

**Discussion by mhazan01** [rspamd/rspamd#4370] [Q&A]
Hello all,

i recently started using rspamd and it's doing an amazing job at keeping the spam and fish out of my users inboxes. i was wondering if there is a plugin or other tools  that could be used to send a summary (ie. from/subject/spam score) of each rejected email to, either the original recipient or a specific email every week or w/e? i've tried to google this but no luck, come up with the dmarc report but that's not what i'm looking for.

Best,

**Comment by schiwe** [comment]
This feature will be nice to have.
